WordPress/src/main/java/org/wordpress/android/ui/media/MediaSettingsActivity.java

```diff
@@ -95,16 +104,24 @@
 
     private SiteModel mSite;
     private MediaModel mMedia;
+    private EditorImageMetaData mEditorImageMetaData;
     private ArrayList<String> mMediaIdList;
+    private String[] mAlignmentKeyArray;
 
     private ImageView mImageView;
     private ImageView mImagePlay;
     private EditText mTitleView;
     private EditText mCaptionView;
     private EditText mAltTextView;
     private EditText mDescriptionView;
+    private EditText mImageWidthView;
+    private SeekBar mImageWidthSeekBarView;
+    private Spinner mAlignmentSpinnerView;
     private FloatingActionButton mFabView;
 
+    private AlertDialog mDeleteImageConfirmationDialog;
```

Comment:
Would it simplify things if instead of two `AlertDialogs` we just had one whose text & action depend on whether the media was shown from the editor?